zip-0320.rst

```diff
@@ -76,7 +76,7 @@ Requirements
    must be able to generate a receiving address such that only transparent
    funds will be spent in transactions with an output to this address.
 2. Wallets and other Consumers that have not been upgraded to recognize the new
-   address format cannot mistake the address for another address type or
+   address format cannot mistake the address for another addRess type or
```

Comment:
Remove this change.

Reply:
Huh, I have no idea how this snuck in. Fat fingers.